gulpfile.babel.js

```diff
@@ -121,7 +127,9 @@ gulp.task('dev', ['dev:doc', 'dev:static', 'dev:applystyles', 'dev:generate'], (
   gulp.watch('lib/styleguide.js', ['dev:generate']);
 });
 
-gulp.task('build', ['sass', 'js:app', 'js:vendor', 'html', 'assets']);
+gulp.task('build', ['clean-dist'], () => {
+  gulp.start(['sass', 'js:app', 'js:vendor', 'html', 'assets']);
```

Comment:
Yeah I just found the same comment. I´m not sure from where I found the start command but clearly it shouldn't be used.